firmware/V3.0/Marlin/Ender 3/bltouch-for-z-homing/Configuration.h

```diff
@@ -1586,7 +1586,7 @@
   // The height can be set with M420 Z<height>
   #define ENABLE_LEVELING_FADE_HEIGHT
   #if ENABLED(ENABLE_LEVELING_FADE_HEIGHT)
-    #define DEFAULT_LEVELING_FADE_HEIGHT 5.0 // (mm) Default fade height.
+    #define DEFAULT_LEVELING_FADE_HEIGHT 0.0 // (mm) Default fade height.
```

Comment:
Did you make this change? I'm wondering if this is an artifact of me updating to the latest version of the code.  The Marlin documentation doesn't even really mention this `DEFAULT_LEVELING_FADE_HEIGHT` setting. That's why I'm questioning whether you made it.

I would think setting this to zero would allow irregularities in the bed to affect the printed piece.

Here's a good [reference link](https://marlinfw.org/docs/configuration/configuration.html#leveling-fade-height) for these settings.

Reply:
Reverting as we don't know where this change came from or why it was made.

Reply:
Reverting as we don't know where this change came from or why it was made.